### PR TITLE
cast fix for settings.server.version.rhel_version for 6.11.z

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
```
==================================================================== 14/5794 tests collected (5780 deselected) in 31.22s ====================================================================
2023-04-27 11:57:11 - robottelo.collection - INFO - Processing test items to add testimony token markers

```

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo

**Review by shweta83 — APPROVED** [review]
Ack

**Review by devendra104 — APPROVED** [review]

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo

**Review by ogajduse — APPROVED** [review]
Not a blocker, but there is no need to cast the string to `int`. Casting it to float is sufficient since `float` can be compared to `int`. 
```
[ins] In [3]: 8.0 == 8
Out[3]: True

[ins] In [4]: 7.9 < 8
Out[4]: True
```

File: tests/foreman/destructive/test_clone.py [hunk 1]
```diff
@@ -27,7 +27,9 @@
 
 
 @pytest.mark.parametrize(
-    "sat_ready_rhel", [7, 8] if settings.server.version.rhel_version < 8 else [8], indirect=True
+    "sat_ready_rhel",
+    [7, 8] if int(float(settings.server.version.rhel_version)) < 8 else [8],
```
Comment (ogajduse) [inline tests/foreman/destructive/test_clone.py:31]: Even better fix would be
```suggestion
    [7, 8] if get_sat_version().major < 8 else [8],
```

Comment (omkarkhatavkar) [reply tests/foreman/destructive/test_clone.py:31]: you mean `get_sat_rhel_version().major`

Comment (omkarkhatavkar) [reply tests/foreman/destructive/test_clone.py]:
```
In [10]: from robottelo.hosts import get_sat_rhel_version

In [11]: get_sat_rhel_version()
Out[11]: <Version('8.7')>

In [12]: get_sat_rhel_version().major
Out[12]: 8

```

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo

**Pull request merged**